requirements.txt

```diff
@@ -7,4 +7,9 @@ pymongo==2.8
 Flask==0.10.1
```

Comment:
And we have some additional requirements like flakes8 to run tests, that not needed for package itself, so they are not present in setup.py